src/Tribe/Admin_List.php

```diff
@@ -21,6 +21,9 @@ class Tribe__Events__Admin_List {
 		public static function init() {
 			if ( is_admin() ) {
 				if ( ! tribe( 'context' )->doing_ajax() ) {
+					// Logic for filtering events by aggregator record
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: c488f52d1f75d84d4a5b7c75b4f7a786</sub>